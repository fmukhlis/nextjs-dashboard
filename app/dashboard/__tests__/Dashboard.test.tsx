import { render, screen } from "@testing-library/react";
import Dashboard from "../page";

describe("DashboardPage", () => {
  it('renders "Dashboard Page" text', () => {
    render(<Dashboard />);
    const dashboardPageText = screen.getByText(/Dashboard Page/i);
    expect(dashboardPageText).toBeInTheDocument();
  });

  // Snapshot
  it("renders DashboardPage unchanged", () => {
    const { container } = render(<Dashboard />);
    expect(container).toMatchSnapshot();
  });
});
