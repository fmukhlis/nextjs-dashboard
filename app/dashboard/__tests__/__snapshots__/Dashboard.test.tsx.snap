// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`DashboardPage renders DashboardPage unchanged 1`] = `
<div>
  <p>
    Dashboard Page
  </p>
</div>
`;
